portfolio/src/main/webapp/index.html

```diff
@@ -10,7 +10,24 @@
     <div id="content">
       <h1>Elijah's Portfolio</h1>
       <h4>Bio</h4>
+       <p> I was born and raised in Chicago, Illinois. I currently live in Logan Square,
```

Comment:
Nit: <p> isn't actually a child element of the headers above, so I don't think it should be indented at all.

Some [example code](https://google.github.io/styleguide/htmlcssguide.html#Separation_of_Concerns) from the style guide shows it aligned with no indentation.